components/molecules/ContributionsTreemap/contributions-treemap.tsx

```diff
@@ -48,31 +56,35 @@ export const ContributionsTreemap = ({ setRepoId, repoId, data, color, onClick }
         <div className="rounded-md overflow-hidden grid place-content-stretch">
           <div className="grid" style={{ gridArea: "1 / 1", minHeight: "29rem" }}>
             <ClientOnly>
-              <ResponsiveTreeMapHtml
-                data={data}
-                tile="squarify"
-                labelSkipSize={12}
-                innerPadding={4}
-                leavesOnly
-                orientLabel={false}
-                nodeComponent={
-                  repoId === null
-                    ? SpecialNode
-                    : // TODO: Sort this out later
-                      (ContributorNode as <Datum extends object>({
-                        node,
-                        animatedProps,
-                        borderWidth,
-                        enableLabel,
-                        labelSkipSize,
-                      }: NodeProps<Datum>) => JSX.Element)
-                }
-                colors={color}
-                nodeOpacity={1}
-                borderWidth={0}
-                onClick={onClick}
-                motionConfig={"default"}
-              />
+              {isLoading ? (
+                <div className="react-loading-skeleton" />
```

Comment:
Should we use our Skeleton Component here?

Reply:
I was going to, but you have to specify a height. I'm going to update the Skeleton loader to not require a height.

I'll follow up with a small PR for that.

Reply:
Actually, I'll add it to this PR, as it is related.

Reply:
👍

Reply:
Done @brandonroberts